cop/docker-compose.yml

```diff
@@ -166,6 +166,7 @@ services:
         ipv6_address: "${DOCKER_MGMT_IP6}5002:4"
     volumes:
       - /home/administrator/.ssh/id_rsa.pub:/app_framework/system_conf/public-key.rsa
+      - /home/administrator/.saml_certs:/app_framework/system_conf/saml_certs
```

Comment:
Can you also add to the volumes to the `cop` container in the `copregion` docker compose file?